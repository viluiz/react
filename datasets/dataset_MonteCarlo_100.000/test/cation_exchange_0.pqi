SOLUTION 0  CaCl2
        units            mmol/kgw
        temp             25.0
        pH               7.0     charge
        pe               12.5    O2(g)   -0.68
        Ca               1.1104192876570247
        Cl               0.46397502606196106
        Na               1.1826684204389684
        K                1.465619864189116
        N(5)             0.019660744682568287                
SOLUTION 1  Initial solution for column
        units            mmol/kgw
        temp             25.0
        pH               7.0     charge
        pe               12.5    O2(g)   -0.68
        Ca               0.7624259667010413
        Cl               0.8764180171711015        
        Na               0.0
        K                1.4694521205414353
        N(5)             0.2830450199550736
EXCHANGE 1
        -equilibrate 1
        X                0.0011
END
