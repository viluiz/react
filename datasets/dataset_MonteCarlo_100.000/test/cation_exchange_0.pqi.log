Input file: cation_exchange_0.pqi

Output file: cation_exchange_0.pqi.out

Database file: phreeqc.dat

              █▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀█
              ║                                            ║
              ║           * PHREEQC-@VERSION@ *            ║
              ║                                            ║
              ║      A hydrogeochemical transport model    ║
              ║                                            ║
              ║                    by                      ║
              ║       D.L. Parkhurst and C.A.J. Appelo     ║
              ║                                            ║
              ║                 @VER_DATE@                 ║
              █▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄█


Initializing...                                                                 

End of Run after 0.011198 Seconds.
