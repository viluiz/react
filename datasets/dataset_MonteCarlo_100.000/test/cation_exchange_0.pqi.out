   Input file: cation_exchange_0.pqi
  Output file: cation_exchange_0.pqi.out
Database file: phreeqc.dat

------------------
Reading data base.
------------------

	SOLUTION_MASTER_SPECIES
	SOLUTION_SPECIES
	PHASES
	EXCHANGE_MASTER_SPECIES
	EXCHANGE_SPECIES
	SURFACE_MASTER_SPECIES
	SURFACE_SPECIES
	RATES
	END
------------------------------------
Reading input data for simulation 1.
------------------------------------

	SOLUTION 0  CaCl2
	        units            mmol/kgw
	        temp             25.0
	        pH               7.0     charge
	        pe               12.5    O2(g)   -0.68
	        Ca               1.1104192876570247
	        Cl               0.46397502606196106
	        Na               1.1826684204389684
	        K                1.465619864189116
	        N(5)             0.019660744682568287                
	SOLUTION 1  Initial solution for column
	        units            mmol/kgw
	        temp             25.0
	        pH               7.0     charge
	        pe               12.5    O2(g)   -0.68
	        Ca               0.7624259667010413
	        Cl               0.8764180171711015        
	        Na               0.0
	        K                1.4694521205414353
	        N(5)             0.2830450199550736
	EXCHANGE 1
	        equilibrate 1
	        X                0.0011
	END
-------------------------------------------
Beginning of initial solution calculations.
-------------------------------------------

Initial solution 0.	CaCl2

-----------------------------Solution composition------------------------------

	Elements           Molality       Moles

	Ca                1.110e-03   1.110e-03
	Cl                4.640e-04   4.640e-04
	K                 1.466e-03   1.466e-03
	N(5)              1.966e-05   1.966e-05
	Na                1.183e-03   1.183e-03

----------------------------Description of solution----------------------------

                                       pH  =  11.595      Charge balance
                                       pe  =   9.031      Equilibrium with O2(g)
      Specific Conductance (µS/cm,  25°C)  = 1138
                          Density (g/cm³)  =   0.99728
                               Volume (L)  =   1.00296
                        Activity of water  =   1.000
                 Ionic strength (mol/kgw)  =   5.871e-03
                       Mass of water (kg)  =   1.000e+00
                 Total alkalinity (eq/kg)  =   4.385e-03
                         Temperature (°C)  =  25.00
                  Electrical balance (eq)  =   4.337e-18
 Percent error, 100*(Cat-|An|)/(Cat+|An|)  =   0.00
                               Iterations  =   9
                                  Total H  = 1.110168e+02
                                  Total O  = 5.551120e+01

----------------------------Distribution of species----------------------------

                                               Log       Log       Log    mole V
   Species          Molality    Activity  Molality  Activity     Gamma    cm³/mol

   OH-             4.331e-03   3.987e-03    -2.363    -2.399    -0.036     -4.06
   H+              2.731e-12   2.538e-12   -11.564   -11.595    -0.032      0.00
   H2O             5.551e+01   9.998e-01     1.744    -0.000     0.000     18.07
Ca            1.110e-03
   Ca+2            1.056e-03   7.687e-04    -2.976    -3.114    -0.138    -18.00
   CaOH+           5.452e-05   5.025e-05    -4.263    -4.299    -0.035     (0)  
Cl            4.640e-04
   Cl-             4.640e-04   4.274e-04    -3.334    -3.369    -0.036     18.12
H(0)          0.000e+00
   H2              0.000e+00   0.000e+00   -44.404   -44.404     0.001     28.61
K             1.466e-03
   K+              1.466e-03   1.350e-03    -2.834    -2.870    -0.036      9.05
N(5)          1.966e-05
   NO3-            1.966e-05   1.808e-05    -4.706    -4.743    -0.036     29.53
Na            1.183e-03
   Na+             1.183e-03   1.091e-03    -2.927    -2.962    -0.035     -1.41
   NaOH            4.345e-16   4.351e-16   -15.362   -15.361     0.001     (0)  
O(0)          5.346e-04
   O2              2.673e-04   2.676e-04    -3.573    -3.572     0.001     30.40

------------------------------Saturation indices-------------------------------

  Phase               SI** log IAP   log K(298 K,   1 atm)

  H2(g)           -41.30    -44.40   -3.10  H2
  H2O(g)           -1.50     -0.00    1.50  H2O
  Halite           -7.90     -6.33    1.57  NaCl
  O2(g)            -0.68     -3.57   -2.89  O2	 Pressure   0.2 atm, phi 1.000
  Sylvite          -7.14     -6.24    0.90  KCl

**For a gas, SI = log10(fugacity). Fugacity = pressure * phi / 1 atm.
  For ideal gases, phi = 1.

Initial solution 1.	Initial solution for column

-----------------------------Solution composition------------------------------

	Elements           Molality       Moles

	Ca                7.624e-04   7.624e-04
	Cl                8.764e-04   8.764e-04
	K                 1.469e-03   1.469e-03
	N(5)              2.830e-04   2.830e-04

----------------------------Description of solution----------------------------

                                       pH  =  11.225      Charge balance
                                       pe  =   9.402      Equilibrium with O2(g)
      Specific Conductance (µS/cm,  25°C)  = 616
                          Density (g/cm³)  =   0.99719
                               Volume (L)  =   1.00299
                        Activity of water  =   1.000
                 Ionic strength (mol/kgw)  =   3.722e-03
                       Mass of water (kg)  =   1.000e+00
                 Total alkalinity (eq/kg)  =   1.835e-03
                         Temperature (°C)  =  25.00
                  Electrical balance (eq)  =  -5.033e-13
 Percent error, 100*(Cat-|An|)/(Cat+|An|)  =  -0.00
                               Iterations  =   6 (15 overall)
                                  Total H  = 1.110143e+02
                                  Total O  = 5.550944e+01

----------------------------Distribution of species----------------------------

                                               Log       Log       Log    mole V
   Species          Molality    Activity  Molality  Activity     Gamma    cm³/mol

   OH-             1.818e-03   1.700e-03    -2.740    -2.770    -0.029     -4.08
   H+              6.326e-12   5.953e-12   -11.199   -11.225    -0.026      0.00
   H2O             5.551e+01   9.999e-01     1.744    -0.000     0.000     18.07
Ca            7.624e-04
   Ca+2            7.453e-04   5.752e-04    -3.128    -3.240    -0.113    -18.04
   CaOH+           1.713e-05   1.603e-05    -4.766    -4.795    -0.029     (0)  
Cl            8.764e-04
   Cl-             8.764e-04   8.199e-04    -3.057    -3.086    -0.029     18.10
H(0)          0.000e+00
   H2              0.000e+00   0.000e+00   -44.404   -44.404     0.000     28.61
K             1.469e-03
   K+              1.469e-03   1.374e-03    -2.833    -2.862    -0.029      9.03
N(5)          2.830e-04
   NO3-            2.830e-04   2.645e-04    -3.548    -3.577    -0.029     29.52
O(0)          5.348e-04
   O2              2.674e-04   2.676e-04    -3.573    -3.572     0.000     30.40

------------------------------Saturation indices-------------------------------

  Phase               SI** log IAP   log K(298 K,   1 atm)

  H2(g)           -41.30    -44.40   -3.10  H2
  H2O(g)           -1.50     -0.00    1.50  H2O
  O2(g)            -0.68     -3.57   -2.89  O2	 Pressure   0.2 atm, phi 1.000
  Sylvite          -6.85     -5.95    0.90  KCl

**For a gas, SI = log10(fugacity). Fugacity = pressure * phi / 1 atm.
  For ideal gases, phi = 1.

-------------------------------------------------------
Beginning of initial exchange-composition calculations.
-------------------------------------------------------

Exchange 1.	

X                1.100e-03 mol

	                               Equiv-    Equivalent      Log 
	Species             Moles      alents      Fraction     Gamma

	CaX2              4.940e-04   9.880e-04   8.982e-01    -0.113
	KX                1.120e-04   1.120e-04   1.018e-01    -0.029

-----------------------------------------
Beginning of batch-reaction calculations.
-----------------------------------------

Reaction step 1.

Using solution 0.	CaCl2
Using exchange 1.	Exchange assemblage after simulation 1.

-----------------------------Exchange composition------------------------------

X                1.100e-03 mol

	                               Equiv-    Equivalent      Log 
	Species             Moles      alents      Fraction     Gamma

	CaX2              4.950e-04   9.900e-04   9.000e-01    -0.138
	KX                9.504e-05   9.504e-05   8.640e-02    -0.036
	NaX               1.494e-05   1.494e-05   1.358e-02    -0.035
	NH4X              7.443e-76   7.443e-76   6.767e-73    -0.037

-----------------------------Solution composition------------------------------

	Elements           Molality       Moles

	Ca                1.109e-03   1.109e-03
	Cl                4.640e-04   4.640e-04
	K                 1.483e-03   1.483e-03
	N                 1.966e-05   1.966e-05
	Na                1.168e-03   1.168e-03

----------------------------Description of solution----------------------------

                                       pH  =  11.595      Charge balance
                                       pe  =   9.031      Adjusted to redox equilibrium
      Specific Conductance (µS/cm,  25°C)  = 1138
                          Density (g/cm³)  =   0.99728
                               Volume (L)  =   1.00296
                        Activity of water  =   1.000
                 Ionic strength (mol/kgw)  =   5.870e-03
                       Mass of water (kg)  =   1.000e+00
                 Total alkalinity (eq/kg)  =   4.385e-03
                         Temperature (°C)  =  25.00
                  Electrical balance (eq)  =  -4.337e-18
 Percent error, 100*(Cat-|An|)/(Cat+|An|)  =  -0.00
                               Iterations  =   4
                                  Total H  = 1.110168e+02
                                  Total O  = 5.551120e+01

----------------------------Distribution of species----------------------------

                                               Log       Log       Log    mole V
   Species          Molality    Activity  Molality  Activity     Gamma    cm³/mol

   OH-             4.331e-03   3.987e-03    -2.363    -2.399    -0.036     -4.06
   H+              2.731e-12   2.538e-12   -11.564   -11.595    -0.032      0.00
   H2O             5.551e+01   9.998e-01     1.744    -0.000     0.000     18.07
Ca            1.109e-03
   Ca+2            1.055e-03   7.680e-04    -2.977    -3.115    -0.138    -18.00
   CaOH+           5.448e-05   5.021e-05    -4.264    -4.299    -0.035     (0)  
Cl            4.640e-04
   Cl-             4.640e-04   4.274e-04    -3.334    -3.369    -0.036     18.12
H(0)          0.000e+00
   H2              0.000e+00   0.000e+00   -44.404   -44.404     0.001     28.61
K             1.483e-03
   K+              1.483e-03   1.365e-03    -2.829    -2.865    -0.036      9.05
N(-3)         0.000e+00
   NH3             0.000e+00   0.000e+00   -71.521   -71.521     0.001     24.42
   NH4+            0.000e+00   0.000e+00   -73.835   -73.872    -0.037     18.00
N(0)          2.724e-32
   N2              1.362e-32   1.364e-32   -31.866   -31.865     0.001     29.29
N(3)          4.072e-18
   NO2-            4.072e-18   3.745e-18   -17.390   -17.427    -0.036     25.01
N(5)          1.966e-05
   NO3-            1.966e-05   1.808e-05    -4.706    -4.743    -0.036     29.53
Na            1.168e-03
   Na+             1.168e-03   1.077e-03    -2.933    -2.968    -0.035     -1.41
   NaOH            4.290e-16   4.296e-16   -15.368   -15.367     0.001     (0)  
O(0)          5.346e-04
   O2              2.673e-04   2.676e-04    -3.573    -3.572     0.001     30.40

------------------------------Saturation indices-------------------------------

  Phase               SI** log IAP   log K(298 K,   1 atm)

  H2(g)           -41.30    -44.40   -3.10  H2
  H2O(g)           -1.50     -0.00    1.50  H2O
  Halite           -7.91     -6.34    1.57  NaCl
  N2(g)           -28.69    -31.87   -3.18  N2
  NH3(g)          -73.32    -71.52    1.80  NH3
  O2(g)            -0.68     -3.57   -2.89  O2
  Sylvite          -7.13     -6.23    0.90  KCl

**For a gas, SI = log10(fugacity). Fugacity = pressure * phi / 1 atm.
  For ideal gases, phi = 1.

------------------
End of simulation.
------------------

------------------------------------
Reading input data for simulation 2.
------------------------------------

----------------------------------
End of Run after 0.011198 Seconds.
----------------------------------

